### ISSUE TITLE
TreeView chevron gets clipped when text scaling is greater than 100%

### ISSUE THREAD

**Issue by kmgallahan** [microsoft/microsoft-ui-xaml#1413]
**Describe the bug**
The TreeView's chevron is clipped when Window's text scaling is set above 100%

**Steps to reproduce the bug**

1. Set text scaling to 125%+
2. Open the XAML Controls Gallery
3. View the TreeView examples

**Expected behavior**
The chevron should not be enlarged and/or clipped.

**Screenshots**
Chevrons at 100% scaling:
![Annotation 2019-10-05 140530](https://user-images.githubusercontent.com/5379488/66259067-c645cb80-e77a-11e9-8413-2e3e8b665824.png)

Chevrons at 125% scaling:
![Annotation 2019-10-05 140611](https://user-images.githubusercontent.com/5379488/66259072-d8276e80-e77a-11e9-8557-ef6c335fcef6.png)

**Version Info**
XAML Controls Gallery version 1.2.10.0

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2019 Update (18362)            | Yes   |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Comment by lukasf** [comment]
Good bug report. The chevron should have `IsTextScaleFactorEnabled="false"`. This is currently missing in the TreeViewItem ControlTemplate, which explains the clipping.

https://docs.microsoft.com/en-us/windows/uwp/design/input/text-scaling#dont-scale-font-based-icons-or-symbols
